hackathon_site/hackathon_site/tests.py

```diff
@@ -69,6 +70,24 @@ def _make_full_registration_team(self, team=None):
 
         return team
 
+    def _review(
+        self,
+        status="Accepted",
+        decision_sent_date=datetime.now().replace(tzinfo=settings.TZ_INFO).date(),
```

Comment:
It's best not to put function calls in function definitions. They only get called once on function creation (in this case, when the test runner first imports the SetupUserMixin), not for every test. This means that things like the `@override_settings` decorator and mocking of `datetime` for one test method will have no effect.

Normally, the way to get around that is to do something like:
```python
def foo(bar=None):
    if bar is None:
        bar = datetime.now()
    ...
```

However in this case we actually want the option for `decsion_sent_date` to be `None` (when no decision has been sent yet), so something like this would work:

```python
def _review(self, status="Accepted", **kwargs):
    decision_sent_date = kwargs.pop("decision_sent_date", datetime.now().replace(tzinfo=settings.TZ_INFO).date()
    ...
```